modules/compliance-profiles.adoc

```diff
@@ -0,0 +1,157 @@
+- rhcos4-selinux-state
+- rhcos4-service-auditd-enabled
+- rhcos4-sshd-disable-empty-passwords
+- rhcos4-sshd-disable-gssapi-auth
+- rhcos4-sshd-disable-rhosts
+- rhcos4-sshd-disable-root-login
+- rhcos4-sshd-disable-user-known-hosts
+- rhcos4-sshd-do-not-permit-user-env
+- rhcos4-sshd-enable-strictmodes
+- rhcos4-sshd-print-last-log
+- rhcos4-sshd-set-loglevel-info
+- rhcos4-sshd-use-priv-separation
+- rhcos4-sysctl-kernel-dmesg-restrict
+- rhcos4-sysctl-kernel-kexec-load-disabled
+- rhcos4-sysctl-kernel-kptr-restrict
+- rhcos4-sysctl-kernel-randomize-va-space
+- rhcos4-sysctl-kernel-unprivileged-bpf-disabled
+- rhcos4-sysctl-kernel-yama-ptrace-scope
+- rhcos4-sysctl-net-core-bpf-jit-harden
+title: Australian Cyber Security Centre (ACSC) Essential Eight
+----
+
+View the rules within a desired profile:
+
+[source,terminal]
+----
+$ oc get -n $NAMESPACE -oyaml rules.compliance <rule name>
+----
+
+.Example output
+[source,yaml]
+----
+apiVersion: compliance.openshift.io/v1alpha1
+description: '<code>auditd</code><code>augenrules</code><code>.rules</code><code>/etc/audit/rules.d</code><pre>-w /var/log/tallylog -p wa -k logins -w /var/run/faillock -p wa -k logins -w /var/log/lastlog -p wa -k logins</pre><code>auditd</code><code>auditctl</code><code>/etc/audit/audit.rules</code><pre>-w /var/log/tallylog -p wa -k logins -w /var/run/faillock -p wa -k logins -w /var/log/lastlog -p wa -k logins</pre>file in order to watch for unattempted manual edits of files involved in storing logon events:'
+id: xccdf_org.ssgproject.content_rule_audit_rules_login_events
+kind: Rule
+metadata:
+  annotations:
+    compliance.openshift.io/rule: audit-rules-login-events
+    control.compliance.openshift.io/NIST-800-53: AU-2(d);AU-12(c);AC-6(9);CM-6(a)
+    policies.open-cluster-management.io/controls: AU-2(d),AU-12(c),AC-6(9),CM-6(a)
+    policies.open-cluster-management.io/standards: NIST-800-53
+    creationTimestamp: "2020-09-07T11:43:03Z"
+    generation: 1
+  labels:
+    compliance.openshift.io/profile-bundle: rhcos4
+  name: rhcos4-audit-rules-login-events
+  namespace: openshift-compliance
+  rationale: |-
+    Manual editing of these files may indicate nefarious activity,
+    such as an attacker attempting to remove evidence of an
+    intrusion.
+  severity: medium
+  title: Record Attempts to Alter Logon and Logout Events
+  warning: |-
+    <ul><li><code>audit_rules_login_events_tallylog</code></li>
+    <li><code>audit_rules_login_events_faillock</code></li>
+    <li><code>audit_rules_login_events_lastlog</code></li></ul>
+    This rule checks for multiple syscalls related to login
+    events and was written with DISA STIG in mind.
+    Other policies should use separate rule for
+    each syscall that needs to be checked.
+----
+
+Each profile has the product it applies prepended to the name. `ocp4-e8` applies the Essential 8 benchmark to the {product-title} product, while `rhcos4-e8` applies the Essential 8 benchmark to the Red Hat CoreOS product.
```

Comment:
IBM Style Guide says not to use "prepend" and to use "add a prefix to" instead.  So maybe:

Each profile had the product name that it applies to added as a prefix to the profile's name.